clang/lib/CodeGen/CGExprComplex.cpp

```diff
@@ -1209,6 +1213,7 @@ LValue ComplexExprEmitter::
 EmitCompoundAssignLValue(const CompoundAssignOperator *E,
           ComplexPairTy (ComplexExprEmitter::*Func)(const BinOpInfo&),
                          RValue &Val) {
+  ApplyAtomGroup Grp(CGF.getDebugInfo());
```

Comment:
This doesn't appear to be symmetrical with the scalar patch, in that `EmitCompoundAssign` gets an atom group for the whole assignment -- should this be the same?

Reply:
>  in that `EmitCompoundAssign` gets an atom group for the whole assignment

I don't think that's true?